google-storage/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "content-provider",
```

Comment:
I notice that there is both a `package-lock.json` and a `yarn.lock` file here. We should only be using one. I'd recommend removing the `yarn.lock` file and just using npm for installs.